pigx-bsseq.in

```diff
@@ -110,6 +110,15 @@ parser.add_argument('--unlock', dest='unlock', action='store_true',
                     help="""\
 Recover after a snakemake crash.""")
 
+parser.add_argument('--verbose', dest='verbose', action='store_true',
+                    help="""\
+Print supp info on job execution.""")
```

Comment:
What is "supp"?  It's better to spell it out; it's a help text after all.

Reply:
fair enough, I'll spell it it out.